core/thread.go

```diff
@@ -441,7 +442,18 @@ func (t *Thread) indexBlock(commit *commitResult, blockType repo.BlockType, targ
 		return err
 	}
 
-	t.pushUpdate(*index)
+	info := &BlockInfo{
+		Id:       index.Id,
+		ThreadId: index.ThreadId,
+		AuthorId: index.AuthorId,
+		Type:     index.Type.Description(),
+		Date:     index.Date,
+		Parents:  index.Parents,
+		Target:   index.Target,
+		Body:     index.Body,
```

Comment:
if you do `t.BlockInfo(index.Id)` you'll get the info w/ `Username` hydrated

Reply:
didn't go this route, but as per our slack convo, merged master and added username to block info